images/adapter/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:stable
+
+RUN apt-get -q -y update && \
+    apt-get install --no-install-recommends -y -q \
+    procps \
+    curl \
+    less \
+    dnsutils \
+    netcat \
+    vim
+
+RUN mkdir -p /opt/logrange/gravity
+
+COPY ["build/adapter", "/opt/logrange/gravity/"]
+
+ENTRYPOINT ["/opt/logrange/gravity/adapter", "start", \
```

Comment:
Can you use `dumb-init`? It will make sure signals are passed to your process correctly.

```
ENTRYPOINT ["/usr/bin/dumb-init", "/opt/logrange/gravity/adapter", ...]
```

`dumb-init` is available inside gravitational/debian-tall.

Reply:
will do

Reply:
applied